projects/angular/src/forms/common/abstract-container.ts

```diff
@@ -129,4 +129,8 @@ export abstract class ClrAbstractContainer implements DynamicWrapper, OnDestroy,
       });
     }
   }
+
+  private get touched() {
+    return this.control && this.control.touched;
```

Comment:
I think this could be `this.control?.touched`.

Reply:
Won't it return `undefined`?
Technically, it's still falsy, but I prefer the clear boolean result.
And I find `!!this.control?.touched` a bit vague.
I agree it's all about preference.
Is there any technical advantage in using one over the other?

Reply:
Both would evaluate to `undefined` if `control` is `undefined`. I'm pretty sure these expressions are exactly the same.

Reply:
And I think returning `undefined` when `control` is `undefined` is correct here. We don't know if the control is touched or not, so `false` could be misleading.

Reply:
I'll change it as you want, because this may be considered better isolation, but just curious, how a control can get undefined after it was touched? Or how it can get touched before it's created.

Reply:
I don't think it can. This code already returns `undefined` when the touched state is unknown. I don't think that should be changed at all. I am only suggesting a code simplification, but it's not a requirement.

Reply:
Done!